### ISSUE TITLE
[core] Use of 0 instead of null (type inference failed)

### ISSUE THREAD

**Issue by bagipro** [skylot/jadx#920]
Hi, class `com.apptentive.android.sdk.comm.ApptentiveClient`
```java
    public static java.lang.String getErrorResponse(java.net.HttpURLConnection httpURLConnection, boolean z) throws java.io.IOException {
        java.io.InputStream inputStream = 0; // <<<<<
        if (httpURLConnection == null) {
            return inputStream; // <<<<<
        }
        try {
            java.io.InputStream errorStream = httpURLConnection.getErrorStream();
            if (errorStream != null && z) {
                errorStream = new java.util.zip.GZIPInputStream(errorStream);
            }
            return com.apptentive.android.sdk.util.Util.readStringFromInputStream(inputStream, "UTF-8");
        } finally {
            com.apptentive.android.sdk.util.Util.ensureClosed(inputStream);
        }
    }
```

APK: https://drive.google.com/file/d/1ICfL4A22K-KhVkw5RwvWdLJA8111iYvG/view?usp=sharing

**Comment by skylot** [comment]
Short problem description:
Now for some try/catch cases constructed incorrect CFG so variables from catch or finally have incorrect scope.
As soon as to fix that CFG need heavy transformation this break lots of other try/catch cases.

**Comment by skylot** [comment]
Fixed in 7c0671c81b1512bf5b56fc4962f89be72d413f27. Although be careful, these changes are big, and definitely break something 🙄

**Comment by bagipro** [comment]
Thanks, @skylot!

I see that they're only a few issues left!
